library/db2

```diff
@@ -55,25 +55,23 @@ EXAMPLES = '''
 - action: db2 instance=my_instance command='db2 list db directory' outputfile=/tmp/output.txt
 - action: db2 instance=my_instance command='db2 list db directory' outputfile=/tmp/output.txt output_format=json
 - action: db2 instance=my_instance dbname=my_cool_db command='db2 get db cfg'
+- action: db2 instance=my_instance dbname=my_cool_db command="db2 attach to my_instance ; db2 get dbm cfg show detail ; db2 detach" outputfile=/tmp/output.txt
 '''
 
+
 def write_to_output(outputfile, output, output_format=False):
     if outputfile and os.path.exists(outputfile) and not outputfile.startswith('/tmp'):
         # Outputfile is not False (so it has been passed), already exists *and*
         # does not start with "/tmp", exit to avoid overwriting scary files
         module.fail_json(msg="outputfile %s already exists and does not reside in /tmp" % outputfile)
     elif outputfile:
         # Outputfile is not false (so it has been passed)
-        f = open(outputfile, 'w')
-        if output_format:
-            output_list = [l.strip() for l in output.split('\n')]
-            while "" in output_list:
-                output_list.remove("")
-
-            json.dump(output_list, f)
-        else:
-            f.write(output)
-        f.close()
+        with open(outputfile, 'w') as f:
+            if output_format:
+                # display the output in json - stdout 'unformatted' way; to format, use .splitlines()
+                json.dump(output, f)
+            else:
+                f.write(output)
```

Comment:
Since I do not have access to db2 anymore: are you sure we don't get empty strings in the output this way? Other than that, I agree this code is a lot cleaner.